css/articles/numeric-types/demos/ex.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="ru">
+<head>
+  <title>ex — Единицы измерения в CSS — Дока</title>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="preconnect" href="https://fonts.gstatic.com/" crossorigin>
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,500;1,400;1,500&display=swap">
+  <style>
+    body {
+      margin: 0;
+      background: #000000;
+    }
+
+    .element {
+      width: 200px;
+      height: 200px;
+      display: flex;
+      align-items: center;
+      justify-content: center;
+      background: #fff;
+    }
+
+    p {
+      margin: 0;
+      font-family: "Roboto", sans-serif;
+      font-size: 24px;
+      width: 100%;
+      text-align: center;
+      position: relative;
+      color: #000;
+    }
```

Comment:
```suggestion
  <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto&display=swap">
  <style>
    body {
      margin: 0;
      background: #000000;
      font-family: "Roboto", sans-serif;
    }

    .element {
      width: 200px;
      height: 200px;
      display: flex;
      align-items: center;
      justify-content: center;
      background: #fff;
    }

    p {
      margin: 0;
      font-size: 24px;
      width: 100%;
      text-align: center;
      position: relative;
      color: #000;
    }
```

Думаю, подход «задать шрифт всей странице» ближе к реальности, плюс не нужны сложные настройки шрифта при подключении.